src/xrGame/ui/UICellItem.cpp

```diff
@@ -48,9 +48,13 @@ CUICellItem::~CUICellItem()
 
 void CUICellItem::init()
 {
-    CUIXml uiXml;
-    if (!uiXml.Load(CONFIG_PATH, UI_PATH, UI_PATH_DEFAULT, "actor_menu_item.xml", false))
-        return;
+    static CUIXml uiXml;
+    static bool isXmlReady = false;
+    if (!isXmlReady)
+    {
+        uiXml.Load(CONFIG_PATH, UI_PATH, UI_PATH_DEFAULT, "actor_menu_item.xml", false);
+        isXmlReady = true;
+    }
```

Comment:
Could you split the changes for this file into separate commit?

Reply:
Done!